ci/test/05_before_script.sh

```diff
@@ -47,6 +47,6 @@ if [ -z "$NO_DEPENDS" ]; then
   fi
   CI_EXEC "$SHELL_OPTS" make "$MAKEJOBS" -C depends HOST="$HOST" "$DEP_OPTS" LOG=1
 fi
-if [ -n "$PREVIOUS_RELEASES_TO_DOWNLOAD" ]; then
-  CI_EXEC test/get_previous_releases.py -b -t "$PREVIOUS_RELEASES_DIR" "${PREVIOUS_RELEASES_TO_DOWNLOAD}"
+if [ -n "$DOWNLOAD_PREVIOUS_RELEASES" ]; then
```

Comment:
This should probably be `if [ "$DOWNLOAD_PREVIOUS_RELEASES" = "true" ]`, otherwise it will run when you set false too.

Reply:
fixed